src/components/book.js

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/jsx-indent */
+/* eslint-disable import/no-cycle */
+/* eslint-disable jsx-a11y/anchor-is-valid */
```

Comment:
- [ ] Kindly ensure that you don't overuse the linter disable. Linters are meant to enforce code quality and too much linter disable means that there is something wrong.